code/modules/projectiles/magazines/revolvers.dm

```diff
@@ -24,6 +24,7 @@
 
 /obj/item/ammo_magazine/revolver/heavy
 	name = "\improper M44 heavy speed loader (.44)"
+	desc = "A revolver speed loader containing heavy bullets. While less damaging overall than the traditional rounds, these are more accurate."
```

Comment:
`desc = "A revolver speed loader containing heavy bullets. While less damaging overall than the traditional rounds, they are more accurate."`

Change these to they, more grammatically accurate